### DISCUSSION TITLE
Overheating using 4K on my Iphone 14 Pro Max

### DISCUSSION THREAD

**Discussion by ysokary1** [qnblackcat/uYouPlus#924] [General]
Hi All my first post so apologies if i have out it in the wrong area!

Anyway, i have been using the uYouPlus app and have realised that when I use 4k on my phone it over heats quiet drastically from the rear i have tested this multiple times. I have watched the same videos on the standard YouTube app and it is absolutely fine. No overheating at all so this seems to be a Uyouplus issue? I have also tested Pip on 1080p and that is fine and then 4k and activating pip while on 4k that does cause the same issue.

Is anyone else experiencing this? Have not seen this discussed before

**Comment by yodaluca23** [comment]
https://github.com/qnblackcat/uYouPlus/issues/6

**Reply by ysokary1** [reply]
Thank you I guess I need to jailbreak the phone whenever that’s possible lol
